app/controllers/user/phone-number/post.controller.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+// Local dependencies
+const { renderErrorView } = require('../../../utils/response')
+const userService = require('../../../services/user_service')
+const paths = require('../../../paths')
+const { invalidTelephoneNumber } = require('../../../utils/validation/telephone-number-validation')
+
+module.exports = async (req, res) => {
```

Comment:
method could be named to describe what the code is doing + show a more useful stack trace

Reply:
👍  async/ await 🎉